tests/test_cases/test_discovery/test_discovery.py

```diff
@@ -89,6 +89,21 @@ async def access_signal(dut):
                          dut.stream_in_data.value.integer, 1))
 
 
+@cocotb.test(skip=cocotb.LANGUAGE in ["vhdl"])
+async def access_type_bit_verilog(dut):
+    """Access type bit in SystemVerilog"""
+    await Timer(1, "step")
+    assert dut.mybit.value == 1, "The default value was incorrect"
+    dut.mybit <= 0
+    await Timer(1, "ns")
+    assert dut.mybit.value == 0, "The assigned value was incorrect"
+
+    assert dut.mybits.value == 0b11, "The default value was incorrect"
+    dut.mybits <= 0b00
+    await Timer(1, "ns")
+    assert dut.mybits.value == 0b00, "The assigned value was incorrect"
```

Comment:
If `bit` doesn't support `X` or `Z`, should there be a test for that? I only ask because I know there are no protections and it will fail in the VPI impl.

Reply:
Good idea, see my comment below.